build.gradle.kts

```diff
@@ -143,22 +143,6 @@ sonar {
     }
 }
 
-allprojects {
-    val projectDir = projectDir.toString()
-    sonar {
-        properties {
-            // Note: folders `kotlin` are not supported (yet), I asked on their side: https://community.sonarsource.com/t/82824
-            // As a workaround provide the path in `sonar.sources` property.
-            if (File("$projectDir/src/main/kotlin").exists()) {
-                property("sonar.sources", "src/main/kotlin")
-            }
-            if (File("$projectDir/src/test/kotlin").exists()) {
-                property("sonar.tests", "src/test/kotlin")
-            }
-        }
-    }
-}
-
```

Comment:
Just to confirm, the work-around is not needed anymore?

Reply:
They recently added support for KMP source sets, so I'd be really surprised if they didn't support the default Kotlin ones 😅 . But yes, the analysis seems to be working fine after removing these lines. I couldn't find where this support was added though.